config/locales/service_providers/es.yml

```diff
@@ -1,6 +1,17 @@
 ---
 es:
   service_providers:
+    errors:
+      generic_sp_link: NOT USED
```

Comment:
this key looks used? is this a temporary value?

Reply:
No, it was a weirdness with the translations making it hard to put a generic string in to replace the link when there wasn't one but I chatted with Annie and we decided to just scrap it and put the generic message for all of them. So this will be going away momentarily.